docs/types.rst

```diff
@@ -847,15 +847,34 @@ Members
         // dynamic array of pairs (i.e. of fixed size arrays of length two).
```

Comment:
This example needs some more love, but probably better to do this after the 0.5.0 release.